props/saber_fett263_buttons.h

```diff
@@ -1725,19 +1731,26 @@ SaberFett263Buttons() : PropBase() {}
             default:
               break;
           }
-          char playtrack[128];
-          RunCommandAndGetSingleLine("list_current_tracks", nullptr, track_num_, playtrack, sizeof(playtrack));
-          track_player_ = GetFreeWavPlayer();
-          if (track_player_) {
-            track_player_->Play(playtrack);
-          } else {
-            STDOUT.println("No available WAV players.");
-          }
+          PlayTrack();
         }
-#else
-        STDOUT.println("Audio disabled.");
-#endif
       }
+    } else {
+      if (track_player_ && !track_player_->isPlaying()) {
+        track_player_.Free();
+      }
+    }
+  }
+
+  void PlayTrack() {
+    char playtrack[128];
+    RunCommandAndGetSingleLine("list_current_tracks", nullptr, track_num_, playtrack, sizeof(playtrack));
+    MountSDCard();
+    EnableAmplifier();
+    track_player_ = GetFreeWavPlayer();
+    if (track_player_) {
+      track_player_->Play(playtrack);
+    } else {
+      STDOUT.println("No available WAV players.");
```

Comment:
If this turns into a common problem, it could be helpful to work around it by checking for available wav players in loop() until one is available and THEN start the playback.

Reply:
I haven't experienced any trouble with get wav players since TrackPlayer can only be entered while saber is off but if you want we can add.  Once track_player_ gets a wav player it keeps until we Free() correct?  I haven't had any reports of trouble but if there are we can add.